src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,235 @@
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(2, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                statement.setLong(1, driver.getId());
+                statement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert drivers to car "
+                    + car + ". ", e);
+        }
+
+    }
+
+    private List<Driver> getDriversForCar(Long carId) {
+        String query = "SELECT id, name, license_number FROM drivers "
+                + "JOIN cars_drivers ON drivers.id = cars_drivers.driver_id "
+                + "WHERE cars_drivers.car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, carId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                drivers.add(parseDriversFromResultSet(resultSet));
+            }
+            return drivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert drivers to car "
+                    + ". ", e);
+        }
+    }
+
+    private Driver parseDriversFromResultSet(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver();
+        driver.setName(resultSet.getString("name"));
+        driver.setLicenseNumber(resultSet.getString("license_number"));
+        driver.setId(resultSet.getObject("id", Long.class));
+        return driver;
+    }
+
+    private Car parseCarWithManufacturerFromResultSet(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setModel(resultSet.getString("model"));
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setName(resultSet.getString("name"));
+        manufacturer.setCountry(resultSet.getString("country"));
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        car.setManufacturer(manufacturer);
+        car.setId(resultSet.getObject("id", Long.class));
+        return car;
+    }
+
+    private Car getCarFromResultSet(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car = parseCarWithDriverIdFromResultSet(resultSet);
+        car.setDrivers(getDriversForCar(car.getId()));
+        return car;
+    }
+
+    private Car parseCarWithDriverIdFromResultSet(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setModel(resultSet.getString("model"));
+        Long manufacturerID = resultSet.getObject("manufacturer_id", Long.class);
+        Manufacturer manufacturer = manufacturerDao.get(manufacturerID).get();
```

Comment:
info about manufacturer should be in result set, you shouldn't get it from `manufacturerDao`